pkg/ibmvpc/analysis_output_test.go

```diff
@@ -345,22 +348,6 @@ var tests = []*vpcGeneralTest{
 		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
 		format:      vpcmodel.MD,
 	},
-	{
-		inputConfig: "acl_testing3",
-		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
-		format:      vpcmodel.Debug,
-	},
-	{
-		inputConfig: "sg_testing1_new",
-		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
-		format:      vpcmodel.Debug,
-	},
-	{
-		inputConfig: "demo_with_instances",
-		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
-		format:      vpcmodel.Debug,
-	},
-	// disable drawio tests until supported with VPE
```

Comment:
please verify these tests exist at least with test output format

Reply:
the same three tests are also in text format